recipes/usgscsm/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+cmake -G "NMake Makefiles" ^
+    -D CMAKE_BUILD_TYPE=Release ^
+    -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -D BUILD_CSM=OFF  ^
```

Comment:
Please add:
```
-D CMAKE_PREFIX_PATH=%PREFIX% ^
-D CMAKE_INSTALL_PREFIX=%PREFIX% ^
```

Reply:
should `-D CMAKE_INSTALL_PREFIX=%PREFIX%` replace `-D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%` or straight up addition? Apologies - weak windows build skills.

Reply:
Sorry, it is ``%LIBRARY_PREFIX%%``

Reply:
For windows is ``%LIBRARY_PREFIX%%`` and for ``linux`` is just ``$PREFIX``